src/components/Combobox/utils/withThrottle.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+
+const rafThrottle = (cb) => {
+  let currentExecution;
+
+  const later = (context, args) => () => {
+    currentExecution = null;
+    cb.apply(context, args);
+  };
+
+  const throttled = (...args) => {
+    if (currentExecution == null) {
+      currentExecution = requestAnimationFrame(later(this, args));
+    }
+  };
+
+  return throttled;
+};
+
+export const withThrottle = Cmp => props => <Cmp {...props} throttleFn={rafThrottle} />;
```

Comment:
`cmp` is commonly used shortcut for `compare`. IMHO better would be `Component` or `Comp`